de.yaml

```
@@ -1,131 +1,131 @@
 2FA_CODE: Code
-2FA_DISABLE: Disable 2FA
-2FA_DISABLE_SUCCESS_MSG: 2FA has been disabled successfully
-2FA_ENABLE: Enable 2FA
-2FA_ENABLE_SUCCESS_MSG: 2FA has been enabled successfully
-2FA_HOW_TO_DISABLE: How to disable 2FA?
-2FA_HOW_TO_DISABLE_STEP_1: Get the 6 digits code on the app you used to enable 2FA and enter it below
-2FA_HOW_TO_DISABLE_STEP_2: Click on the button!
-2FA_HOW_TO_DISABLE_STEP_3: Delete the NosWings:YourUsername key on the app to be able to scan the code again.
-2FA_HOW_TO_ENABLE: How to enable 2FA?
-2FA_HOW_TO_ENABLE_STEP_1:  Download an 2FA Authenticator App
-2FA_HOW_TO_ENABLE_STEP_2: Scan the QR Code using your application previously downloaded.
-2FA_HOW_TO_ENABLE_STEP_3: You will have a 6 digits code. Enter it below.
-2FA_HOW_TO_ENABLE_STEP_4: Click on the button!
-2FA_MANAGE: Manage 2FA
-ABOUT: About
-ACCOUNT: Account
-ACCOUNT_ALREADY_CREATED: Already have an account?
-ACCOUNT_CREATE: Create an account
-ACCOUNT_CREATED: Account created!
-ACCOUNT_ID: Account ID
-ACCOUNT_SUCCESSFULLY_ACTIVATED: Your account has been successfully activated! Login below.
-ACCOUNT_VERIFICATION_LINK_ERROR: An error occurred. Please verify your activation link.
+2FA_DISABLE: 2FA deaktivieren
+2FA_DISABLE_SUCCESS_MSG: 2FA wurde erfolgreich deaktiviert
+2FA_ENABLE: 2FA aktivieren
+2FA_ENABLE_SUCCESS_MSG: 2FA wurde erfolgreich aktiviert
+2FA_HOW_TO_DISABLE: Wie deaktiviere ich die 2FA?
+2FA_HOW_TO_DISABLE_STEP_1: Trage den 6-stelligen Code aus deiner für die Aktivierung von 2FA genutzten App unten ein.
```

```suggestion
2FA_HOW_TO_DISABLE_STEP_1: Trage den 6-stelligen Code aus deiner 2FA App unten ein.
```
mMn. ein unnötig langer Satz

---

Stimme ich zu.